storage/tests/block_storage.rs

```diff
@@ -57,22 +54,6 @@ fn test_put_block_header_twice() -> Result<(), Error> {
     Ok(())
 }
 
-#[test]
```

Comment:
@binier aren't we going to do this anymore?

Reply:
I don't know if it is used for anything right now, I think you mentioned that it wasn't, just in these tests. But it is useful to do this to be able to check the consistency between the main db and the context storage at startup.